runtime/protocol/http/common/src/aws/smithy/kotlin/runtime/http/endpoints/functions/Functions.kt

```diff
@@ -2,7 +2,10 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0.
  */
-package aws.smithy.kotlin.runtime.http.endpoints.util
+/**
+ * This package implements common standard library functions used by endpoint resolvers.
+ */
+package aws.smithy.kotlin.runtime.http.endpoints.functions
```

Comment:
**Comment**: I don't believe this actually applies KDoc to the package. See https://kotlinlang.org/docs/kotlin-doc.html#module-and-package-documentation for more detail. I suggest you either:
* Move it to a separate markdown file if the intended audience is users
* Change this to a non-doc comment if the intended audience is only SDK code authors
* Delete it if this package doesn't warrant its own documentation